quick_start.sh

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash
+set -euo pipefail
+
+# Colors and styles
+RESET='\033[0m'
+BOLD='\033[1m'
+
+# Use colors if we can and have the color space
+if command -v tput &> /dev/null; then
+    ncolors=$(tput colors)
+    if [ -n "$ncolors" ] && [ "$ncolors" -ge 8 ]; then
+        RESET=$(tput sgr0)
+        BOLD=$(tput bold)
+    fi
+fi
+
+print_message() {
+    local color="$1"
+    local message="$2"
+    local format="${3:-}"
+    echo -e "${format}${color}${message}${RESET}"
+}
+
+print_partial_message() {
+    local pre_message="$1"
+    local formatted_part="$2"
+    local post_message="$3"
+    local format="$4"
+    local color="${5:-$RESET}"
+
+    echo -e "${color}${pre_message}${format}${formatted_part}${RESET}${color}${post_message}${RESET}"
+}
+
+print_message "" "" ""
+print_message "" "   ╭───────────────────────────────────╮" ""
+print_message "" "   │  Welcome to the Warnet Quickstart │" ""
+print_message "" "   ╰───────────────────────────────────╯" ""
+print_message "" "" ""
+print_message "" "    Let's find out if your system has what it takes to run Warnet..." ""
+print_message "" "" ""
+
+current_git=$(basename "$(git rev-parse --show-toplevel)" || true)
+if [ -n "$current_git" ]; then
+    print_message "" " ⭐️ Currently in the warnet directory"
+else
+    print_message "" " Please navigate to the warnet directory."
+    exit 1
+fi
+
+minikube_path=$(command -v minikube || true)
+if [ -n "$minikube_path" ]; then
+    print_partial_message " ⭐️ Found " "minikube" ": $minikube_path " "$BOLD"
+else
+    print_partial_message " 💥 Could not find " "minikube" ". Please follow this link to install it..." "$BOLD"
+    print_message "" "   https://minikube.sigs.k8s.io/docs/start/" "$BOLD"
+    exit 127
+fi
+
+kubectl_path=$(command -v kubectl || true)
+if [ -n "$kubectl_path" ]; then
+    print_partial_message " ⭐️ Found " "kubectl" ": $kubectl_path " "$BOLD"
+else
+    print_partial_message " 💥 Could not find " "kubectl" ". Please follow this link to install it..." "$BOLD"
+    print_message "" "   https://kubernetes.io/docs/tasks/tools/" "$BOLD"
+    exit 127
+fi
+
+docker_path=$(command -v docker || true)
+if [ -n "$docker_path" ]; then
+    print_partial_message " ⭐️ Found " "docker" ": $docker_path" "$BOLD"
+else
+    print_partial_message " 💥 Could not find " "docker" ". Please follow this link to install it..." "$BOLD"
+    print_message "" "   https://minikube.sigs.k8s.io/docs/drivers/docker/" "$BOLD"
```

Comment:
I don't particuarly mind linking to this page as it has a link to docker docs, but this could also just directly lilnk to https://docs.docker.com/engine/install/ ?

Also, should we recommend installing "Docker Enginer" and _not_ "Docker Desktop"? (probably not, I'm prob just a CLI snob...)